pro/cypress/e2e/step-definitions/financialManagement.cy.ts

```diff
@@ -52,3 +52,48 @@ Then(
     })
   }
 )
+
+Then('These receipt results should be displayed', (dataTable: DataTable) => {
+  const numRows = dataTable.rows().length
+  const numColumns = dataTable.raw()[0].length
+  const data = dataTable.raw()
+
+  cy.findAllByTestId('spinner').should('not.exist')
+  cy.findAllByTestId('invoice-item-row').should('have.length', numRows)
+
+  // Vérification des titres des colonnes
+  const titleArray = data[0]
+  cy.findAllByTestId('invoice-title-row').within(() => {
+    cy.get('th').then(($elt) => {
```

Comment:
c'est quoi elt?

Reply:
Une variable, ici l'élément yielded (comme ils disent ici: https://docs.cypress.io/api/commands/then#The-button-element-is-yielded) donc la partie du dom sous le <th>

Reply:
J'ai réorganisé les 3 tests en 1 seul. A noter qu'à cause du cloudfare vérification qu'on est pas un robot, je suis obligé de forcer la visite de l'url après avoir cliquer sur un des liens externes.